assets/Two factor.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 89.71 135.25"><defs><style>.cls-1,.cls-2{fill:#fff;}.cls-1,.cls-3{stroke:#000;stroke-miterlimit:10;stroke-width:3px;}.cls-3{fill:none;}</style></defs><title>Two factor</title><path class="cls-1" d="M126.35,60.33a29.36,29.36,0,0,0-58.59,0h-.08v44.22h58.74V60.33Z" transform="translate(-51.75 -31.49)"/><path class="cls-2" d="M96.6,165.24A43.36,43.36,0,1,1,140,121.88,43.4,43.4,0,0,1,96.6,165.24Z" transform="translate(-51.75 -31.49)"/><path d="M96.6,80a41.86,41.86,0,1,1-41.85,41.85A41.9,41.9,0,0,1,96.6,80m0-3a44.86,44.86,0,1,0,44.86,44.85A44.85,44.85,0,0,0,96.6,77Z" transform="translate(-51.75 -31.49)"/><rect class="cls-3" x="18.71" y="82.01" width="51.92" height="16.98" rx="6.57"/><circle cx="44.81" cy="90.22" r="4.04"/><circle cx="61.03" cy="90.23" r="4.04"/><circle cx="28.85" cy="90.25" r="4.04"/></svg>
```

Comment:
Did you create the icon yourself or is it being re-used from somewhere?

Could we rename it to `two-factor.svg` to match the rest of the files?

Reply:
I've got the inspiration by searching for `vault`, `password`, `key` icons. And based on that I created it myself in illustrator so there's no license. So if you need an official approval, you have mine to use it for whatever reason for this project.

Reply:
Thanks for the confirmation! The plugin is released under the GPL license so the assets would inherit it unless you release it under another license. It just [has to be compatible with GPL](https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/#1-plugins-must-be-compatible-with-the-gnu-general-public-license).